Zotero/Scenes/Detail/Annotation Popover/Views/ColorPickerCell.swift

```diff
@@ -11,42 +11,66 @@ import UIKit
 import RxSwift
 
 final class ColorPickerCell: UITableViewCell {
-    @IBOutlet private weak var stackView: UIStackView!
+    let colorChange: PublishSubject<String>
+    let disposeBag: DisposeBag
```

Comment:
Doesn't this `disposeBag` need to be disposed on cell reuse? I guess there's always only one used, but just to be sure in case we use this somewhere else.

Reply:
Check https://github.com/zotero/zotero-ios/pull/807/commits/07b7b802048eca7e64129908c684f0be3e3a0300

I see that there is the `RxTableViewCell` class. Perhaps the same logic should apply there, that is upon reuse replace the `disposeBag` (and therefore dispose the old as it is deinitialized), rather than expecting `newDisposeBag` to be called explictly?

Reply:
@mvasilak sure, I think the same should apply there. You can try changing it and test whether everything works as expected.

Reply:
Check https://github.com/zotero/zotero-ios/pull/807/commits/70f5ff7c73a63499f9093c63eb82caf098f75da6
Even if there is only one cell, it might be reused if it is not visible due to scrolling, and then is setup again. So it's indeed better to properly prepare for reuse.